tests/models/test_user.py

```
@@ -20,10 +20,10 @@ def tearDown(self):
 
```

In future, we need to change `raise Exception('Not valid git email form')` into `raise InvalidEmailException` something like that 👍